pkg/manifest/properties_hash.go

```diff
@@ -0,0 +1,64 @@
+package manifest
+
+import "github.com/pkg/errors"
+
+type HashAlgorithm string
+
+// The following hashes keys are reserved for future use, but not necessarily supported by the toolkit.
+// If you are using a hash algorithm not listed here, it's better to use a URI, such as `https://blurha.sh`.
+// If there's an algorithm you think should be recognized, let us know.
+const (
+	HashAlgorithmBlake2b  HashAlgorithm = "blake2b"
```

Comment:
Isn't blur.sh supported by the toolkit? Why is it private and not part of this enum?

Reply:
@mickael-menu While it's supported by the toolkit, I didn't want to "immortalize" it and make it officially supported until @HadrienGardeur had a chance to think about this. His initial request was to support hashing of the file (thus the well-known hashing algorithms in that enum), as well as a perceptual hash, for which I chose `phash-dct`. The addition of blurhash was a bonus you requested, but I'm not sure if we should turn it into a "hash algorithm" since that's not really what it is, because the value is an actual thumbnail-sized representation of the image which cannot really be mathematically compared to another hash for equivalency or similarity. Let me know if you think differently